housing_django/templates/login.html

```diff
@@ -0,0 +1,62 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  {% if form.errors %}
+    <p>Your username and password didn't match. Please try again.</p>
+  {% endif %}
+  {% if next %}
+    {% if user.is_authenticated %}
+
+<p>Your account doesn't have access to this page. To proceed,
+    please login with an account that has access.</p>
+    {% else %}
+
+<p>Please login to see this page.</p>
+    {% endif %}
+{% endif %}
+<div class="container">
+  <div class="row">
+    <div class="col-md-4 col-md-offset-4">
+      <div class="login-panel panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title">Please Sign In</h3>
+        </div>
+        <div class="panel-body">
+          <form method="post" action="{% url 'login' %}">
+            {% csrf_token %}
+            <p class="bs-component">
```

Comment:
Actually, maybe you can remove the <p> altogether. It doesn't make much sense to be wrapping the form inputs.